uv.lock

```diff
@@ -1,6 +1,10 @@
 version = 1
-revision = 3
+revision = 2
```

Comment:
Could it be that you are running an old version of `uv` and that is downgrading the file format? I am guessing here but I think I had this issue myself at some point.

Try updating your `uv`, then resetting the uv lock file to `main` and then rerun the `uv` commands... It shouldn't change 2.8k lines.